### PR TITLE
Feat: nixbsd

### LINKED ISSUE #246: Feature Request : nixbsd
https://github.com/nixos-bsd/nixbsd

### PR DESCRIPTION
**Pull request by Malix-off**
Closes #246

[Would probably be better bellow SnowflakeOS](https://github.com/nix-community/awesome-nix/pull/248)

### CODE REVIEW

**Review by cyntheticfox — CHANGES_REQUESTED** [review]
Aside from the lint failing from the lack of punctuation, the only real issue I see is that this should probably be above NixNG to maintain case-insensitive alphabetical ordering. I'm pretty sure that would also fix any potential merge conflicts from the SnowflakeOS one

File: README.md [hunk 1]
```diff
@@ -294,6 +294,7 @@
 ## Distributions
 
 * [NixNG](https://github.com/nix-community/NixNG) - A GNU/Linux distribution similar to NixOS, defining difference is a focus on containers and lightweightness.
+* [nixbsd](https://github.com/nixos-bsd/nixbsd) - A NixOS fork with a FreeBSD kernel
```
Comment (Malix-off) [inline README.md:297]:
```suggestion
* [nixbsd](https://github.com/nixos-bsd/nixbsd) - A NixOS fork with a FreeBSD kernel.
```

**Comment by Malix-off** [comment]
>the only real issue I see is that this should probably be above NixNG

Agreed

**Review by cyntheticfox — COMMENTED** [review]
Looks good!

**Review by cyntheticfox — APPROVED** [review]
Hit the wrong option in the app (comment instead of approve), so here we go

**Pull request merged**